src/components/Services/DtsList.tsx

```diff
@@ -185,6 +233,52 @@ function DtsList() {
     );
   }
 
+  async function getDtscCollection(collectionFk: string): Promise<DtscCollectionVO> {
+    const api = createDtsResourceApi();
+    const collecFk: DtscGetIdGetRequest = { id: collectionFk };
+    return api.dtscGetIdGet(collecFk);
+  }
+
+  async function templateData(collectionFk: string, idService: string): Promise<TemplateDataInfo> {
+    try {
+      const response = await getDtscCollection(collectionFk);
+      return {
+        idService: idService,
+        idCollection: collectionFk,
+        templateRepo: response.templateRepo,
+        template: response.template,
+      };
+    } catch (error) {
+      return {
+        idService: 'undefined',
+        idCollection: 'undefined',
+        templateRepo: "no repository",
+        template: "no template",
+      };
+    }
+  }
+
+  async function getTemplatesInfoData(dtsVOs: DtsVO[]): Promise<void> {
+    const templatePromises = dtsVOs.map((dts) =>
+      templateData(String(dts.collectionFk), String(dts.id))
+    );
+
+    const templDatasInfo = await Promise.all(templatePromises);
+
+    if (!templatesDatasInfo) {
+      setTemplatesDatasInfo(templDatasInfo);
+    }
+  }
+
+  function getTemplateDataInfo(idService: string): TemplateDataInfo {
+    return (
+      templatesDatasInfo?.find(
```

Comment:
in generic value use default
{ idService: 'service', idCollection: 'uuid', templateRepo: 'no repository', template: 'no template' }

Note: remember that 'undefined' may scare the user